mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -55,40 +55,11 @@ class _DashboardViewState extends State<DashboardView>
     return Scaffold(
       appBar: PreferredSize(
         preferredSize: const Size.fromHeight(60.0),
-        child: Showcase.withWidget(
-          key: _skipShowcaseKey,
-          overlayOpacity: 0.6,
-          targetShapeBorder: const RoundedRectangleBorder(),
-          width: 50,
-          height: 50,
-          container: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: <Widget>[
-              Container(
-                width: 45,
-                height: 45,
-                decoration: const BoxDecoration(
-                  shape: BoxShape.circle,
-                  color: Colors.white,
-                ),
-                child: IconButton(
-                  tooltip: "Skip Showcase",
-                  icon: const Icon(Icons.skip_next),
-                  onPressed: () => ShowCaseWidget.of(context).dismiss(),
-                  color: CustomColors.appColorBlue,
-                ),
-              ),
-              const SizedBox(
-                height: 10,
-              ),
-              const Text(
-                "Click to Skip Tutorial",
-                textAlign: TextAlign.left,
-                style: TextStyle(color: Colors.white),
-              ),
-            ],
-          ),
-          child: AppBar(
+        child: CustomShowcaseWidget(
+          showcaseKey: _skipShowcaseKey,
+          description: "Click to Skip Tutorial",
+          customize: "skip",
```

Comment:
its better to have `skip` and `up` stored in an enum.

Reply:
Done